soda-data-quality-checks/Makefile

```diff
@@ -0,0 +1,17 @@
+usage:		## Shows usage for this Makefile
+	@cat Makefile | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
+
+install:    ## Install dependencies
+	which awslocal || pip install 'awscli-local[ver1]'
+	which soda || pip install soda-core-snowflake
```

Comment:
Providing: `which snow || pip install snowflake-cli-labs` may help during the installation of deps, since we are using that command afterwards.